src/dsp_tools/utils/project_create.py

```diff
@@ -744,11 +744,6 @@ def _add_cardinalities_to_resource_classes(
             if ":" in card_info["propname"]:
                 prefix, prop = card_info["propname"].split(":")
                 qualified_propname = card_info["propname"] if prefix else f"{ontology_remote.name}:{prop}"
-                if not remote_prop_classes.get(ontology_remote.id + "#" + prop):
```

Comment:
This validation is misleading and unnecessary.
Until now, it worked because of a peculiarity of the test onto. But now that I modified the test onto, this validation became a handicap: `remote_prop_classes` doesn't contain properties from other ontos from the same JSON file. 
I realised that the validation is unnecessary, because some lines further down, an attempt is made to add the cardinality to the resource class, and if the attempt is not successful, the same warning will be printed.